src/main/java/com/example/demo/Email.java

```diff
@@ -0,0 +1,102 @@
+package com.example.demo;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.sql.SQLOutput;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+
+@Component
+public class Email implements Serializable {
```

Comment:
Stateful classes (that should hold data and are mutable) are bad candidates for being spring beans. Sometimes it's worth it but not in this case. You don't need a dependencies and you're creating instances of this class yourself (with new keyword, as any other class). There is no benefit of making this as spring component